File: backend/DocCareWeb/src/DocCareWeb.Domain/Entities/Address.cs
namespace DocCareWeb.Domain.Entities
{
    public class Address : BaseEntity
    {
        public int PatientId { get; set; }
        public string Street { get; set; } = string.Empty;
        public string? Number { get; set; }
        public string? Complement { get; set; }
        public required string City { get; set; }
        public string? Neighborhood { get; set; }        
        public  string State { get; set; } = string.Empty;
        public string ZipCode { get; set; } = string.Empty;
    }
}

File: frontend/doccareweb/src/app/features/patient/models/address.ts
export interface Address {
    id?: number;
    street: string;
    number: string;
    complement?: string;
    neighborhood: string;
    city: string;
    state: string;
    zipCode: string;
}

export interface SearchZipCode {
    cep: string;
    logradouro: string;
    complemento: string;
    bairro: string;
    localidade: string;
    uf: string;
}
